### DISCUSSION TITLE
已经续费1年，没有拿到验证码

### DISCUSSION THREAD

**Discussion by clarkpkchyg** [getlantern/lantern-forum-cn#768] [General]
已经发邮件给mike：mike@getlantern.org
购买记录截图也已附件发送，盼复

**Comment by MaxMKMax** [comment]
@clarkpkchyg 好的. 谢谢您. 我帮您提交上去.

**Reply by clarkpkchyg** [reply]
尊敬的用户，
我们很遗憾的通知您，您发的邮件中有一个或多个接收人没有投递成功。
退回的邮件在附件中，请查看附件了解详情。
被退回邮件信息：
服务器：59.24.3.173
退信原因：Connect to remote host error.
收件人：mike@getlantern.org
退信原因查询：http://help.sina.com.cn/i/90/152_2.html
我感觉新浪邮箱发不出去

**Comment by clarkpkchyg** [comment]
不知道为什么新浪邮箱发不出去邮件
***@***.***
***@***.***

Mike K ***@***.***> 于2024年8月8日周四 23:20写道：

> @clarkpkchyg <https://github.com/clarkpkchyg> 好的. 谢谢您. 我帮您提交上去.
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/getlantern/lantern-forum-cn/discussions/768#discussioncomment-10277002>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/BKM5JFCVT62WIO4HGYTMCO3ZQOEEHAVCNFSM6AAAAABMGROW5OVHI2DSMVQWIX3LMV43URDJONRXK43TNFXW4Q3PNVWWK3TUHMYTAMRXG4YDAMQ>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***
> .com>
>


-- 
陈亚光
首都经济贸易大学外语学院 本科三年级
***@***.***
联系方式：86-13681252758
Yaguang Chen
Capital University of Business and Economics
Foreign Language Department
BA in Business English
Secretary-General, CUEB Model United Nations Association
Cell:86-13681252758
***@***.***
